lib/diagnostic.rb

```diff
@@ -45,7 +57,11 @@
 # which returns the string "think think think".
 
 ##
-# your response here
+class Developer < Person
+  def solve_problems
+    puts 'think think think'
```

Comment:
This will return `nil` since `puts` doesn't return anything.  You can use an explicit return with `return 'think think 'think'` or an implicit return with `'think think think'`.